op-devstack/dsl/l2_network.go

```diff
@@ -252,3 +253,7 @@ func (n *L2Network) DisputeGameFactoryProxyAddr() common.Address {
 func (n *L2Network) DepositContractAddr() common.Address {
 	return n.inner.RollupConfig().DepositContractAddress
 }
+
+func (n *L2Network) L1ScalarMultiplierConfig() (*big.Int, *big.Int) {
+	return n.inner.RollupConfig().ChainOpConfig.L1ScalarMultipliers(0)
```

Comment:
The `L1ScalarMultipliers` method should be called with the corresponding block time.